stock_mts_mto_rule/tests/test_mto_mts_route.py

```diff
@@ -94,6 +129,52 @@ def test_mts_mto_route_split(self):
         self.assertEqual(1, len(move_mto))
         self.assertEqual("waiting", move_mto.state)
 
+    def test_mts_mto_route_split_no_group(self):
+        mto_mts_route = self.env.ref("stock_mts_mto_rule.route_mto_mts")
+        self.product.route_ids = [(6, 0, [mto_mts_route.id])]
+        self._create_quant(1.0)
+        vals = self.procurement_vals.copy()
+        vals.pop("group_id")
+        self.env["procurement.group"].run(
+            [
+                self.group.Procurement(
+                    self.product,
+                    2.0,
+                    self.uom,
+                    self.customer_loc,
+                    self.product.name,
+                    "test",
+                    self.warehouse.company_id,
+                    vals,
+                )
+            ]
+        )
+        moves = self.env["stock.move"].search(
+            [("product_id", "=", self.product.id), ("group_id", "=", False)]
+        )
+        self.assertEqual(3, len(moves))
```

Comment:
I wonder if the test would not be clearer if we add a `('state', '!=', 'done')` in the domain and then test that we have 2 moves. Because knowing the feature of this module, I know I run a procurement and expect 2 moves instead of 1, I though it was a bit confusing the have a test of a number of moves that is not really related to what we test, just because we had to make an inventory !
Well, it is really a minor comment, it is not blocking at all.